### PR TITLE
Rename CollectionType entry options

### PR DESCRIPTION
**Pull request by wouterj**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes (https://github.com/symfony/symfony/pull/15025) |
| Applies to | 2.7+ |
| Fixed tickets | - |

### CODE REVIEW

File: reference/forms/types/collection.rst [hunk 1]
```diff
@@ -16,10 +16,10 @@ forms, which is useful when creating forms that expose one-to-many relationships
 | Options     | - `allow_add`_                                                              |
 |             | - `allow_delete`_                                                           |
 |             | - `delete_empty`_                                                           |
-|             | - `options`_                                                                |
+|             | - `entry_options`_                                                                |
```
Comment (xabbuh) [inline reference/forms/types/collection.rst]: Shouldn't we keep the old options marking them as deprecated? Removing them entirely can be confusing for people coming from older versions.

Comment (xabbuh) [inline reference/forms/types/collection.rst]: By the way, you to realign the pipe character (same below).

Comment (WouterJ) [inline reference/forms/types/collection.rst]: we've never kept deprecated options afaik

File: reference/forms/types/collection.rst [hunk 2]
```diff
@@ -16,10 +16,10 @@ forms, which is useful when creating forms that expose one-to-many relationships
 | Options     | - `allow_add`_                                                              |
 |             | - `allow_delete`_                                                           |
 |             | - `delete_empty`_                                                           |
-|             | - `options`_                                                                |
+|             | - `entry_options`_                                                          |
+|             | - `entry_type`_                                                             |
 |             | - `prototype`_                                                              |
 |             | - `prototype_name`_                                                         |
-|             | - `type`_                                                                   |
```
Comment (xabbuh) [inline reference/forms/types/collection.rst]: You have to change the option name in line 14 too.

**Comment by weaverryan** [comment]
Congrats @WouterJ on the merge! I've merged this into 2.8 and (hopefully) fixed the build failure.

**Pull request closed without merging**